platform/security/core/security-core-services/src/main/java/ddf/security/http/impl/HttpSessionFactory.java

```diff
@@ -44,4 +50,13 @@ public synchronized HttpSession getOrCreateSession(HttpServletRequest httpReques
     }
     return session;
   }
+
+  public void setExpirationTime(int expirationTime) {
+    // Sets expirationTime to the default if the provided value is less than 2
+    if (expirationTime >= 2) {
+      this.expirationTime = expirationTime;
+    } else {
+      this.expirationTime = DEFAULT_EXPIRATION_TIME;
```

Comment:
❓ Why not set this to `2` instead of the default?

Reply:
To be clear, I'm just maintaining the existing behavior here. The main goal of this PR is to move the logic to a more central location, so that there's no change that a session will get created and not follow this logic.

That being said, I believe this has been defaulted to 30 because having a fairly low value introduces a lot of overhead around maintaining sessions and can also cause possible race conditions.